packages/playwright-test/src/runner/loadUtils.ts

```diff
@@ -219,13 +221,15 @@ async function createProjectSuite(fileSuites: Suite[], project: FullProjectInter
   return projectSuite;
 }
 
-function createForbidOnlyErrors(onlyTestsAndSuites: (TestCase | Suite)[]): TestError[] {
+function createForbidOnlyErrors(onlyTestsAndSuites: (TestCase | Suite)[], forbidOnlyCLIFlag: boolean | undefined, configFilePath: string | undefined): TestError[] {
   const errors: TestError[] = [];
   for (const testOrSuite of onlyTestsAndSuites) {
     // Skip root and file.
-    const title = testOrSuite.titlePath().slice(2).join(' ');
+    const title = testOrSuite.titlePath().slice(2).join(' ').replaceAll(path.sep, '/');
```

Comment:
I added this to make the Windows test work. Not sure why it was working before?

Reply:
We should leave user-visible path as is, and adjust the test instead. On Windows, I'd like to see `C:\Users...` instead of unix-style paths. [Example here](https://github.com/microsoft/playwright/blob/main/tests/playwright-test/runner.spec.ts#LL34C29-L34C29).

Reply:
Sorry, I resolved this by accident before. I have reverted this but not sure how to adjust the test - in only happens in the Windows CI run but the test code is shared right?